db/migrate/20170427152006_remove_invalid_hawkular_endpoints.rb

```diff
@@ -0,0 +1,29 @@
+class RemoveInvalidHawkularEndpoints < ActiveRecord::Migration[5.0]
+  class ExtManagementSystem < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  class Endpoint < ActiveRecord::Base; end
+
+  def up
+    Endpoint.where(
+      :resource_type => "ExtManagementSystem",
+      :role          => 'hawkular',
+      :hostname      => nil
```

Comment:
nil in the bug description[1] but it's a good idea to test both options

[1] https://bugzilla.redhat.com/show_bug.cgi?id=1437138#c19